### PR TITLE
Bug 1180245. Moved php sites into pontoon

### PR DESCRIPTION
**Pull request by jotes**
@mathjazz hi, could you review?

### CODE REVIEW

**Comment by mathjazz** [comment]
Excellent job, @jotes!

I've only found one bug - this page raises 'ascii' codec can't decode byte error:
http://localhost:8000/sites/tiles/?file=suggestedtiles_infographic.lang

A few nits:
- Move static/snippets/ files into css/ and images/ subfolders.
- Move tiles/css/whimsycorn.png to tiles/images/whimsycorn.png.
- Move updater inline CSS into external static CSS file.

File: pontoon/sites/urls.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    url(r'^tiles/$', views.sites_snippet_page,
+        kwargs=dict(template="tiles.html",
+                    repository_url="https://svn.mozilla.org/projects/l10n-misc/trunk/firefoxtiles/en-US/",
+                    default_filename="tiles.lang"),
+        name='sites-tiles'),
+    url(r'^snippets/$', views.sites_snippet_page,
+        kwargs=dict(template="snippets.html",
+                    repository_url="https://svn.mozilla.org/projects/l10n-misc/trunk/snippets/en-US/",
+                    default_filename="jan2014.lang"),
+        name='sites-tiles'),
```
Comment (mathjazz) [inline pontoon/sites/urls.py]: sites-snippets

Comment (jotes) [inline pontoon/sites/urls.py]: Okay, i'll fix these bugs tomorrow. Thx for review.

On Mon, Aug 31, 2015 at 8:02 PM, Matjaž Horvat <notifications@github.com>
wrote:

> In pontoon/sites/urls.py
> <https://github.com/mozilla/pontoon/pull/137#discussion_r38339703>:
>
> > @@ -0,0 +1,21 @@
> > +from django.conf.urls import patterns, url
> > +import views
> > +
> > +urlpatterns = patterns('',
> > +    # Sites pages
> > +    url(r'^tiles/$', views.sites_snippet_page,
> > +        kwargs=dict(template="tiles.html",
> > +                    repository_url="https://svn.mozilla.org/projects/l10n-misc/trunk/firefoxtiles/en-US/",
> > +                    default_filename="tiles.lang"),
> > +        name='sites-tiles'),
> > +    url(r'^snippets/$', views.sites_snippet_page,
> > +        kwargs=dict(template="snippets.html",
> > +                    repository_url="https://svn.mozilla.org/projects/l10n-misc/trunk/snippets/en-US/",
> > +                    default_filename="jan2014.lang"),
> > +        name='sites-tiles'),
>
> sites-snippets
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/pontoon/pull/137/files#r38339703>.
>



-- 
jarek@reijutsu:~$ fortune
You have Egyptian flu: you're going to be a mummy.
jarek@reijutsu:~$ fortune
You now have Asian Flu.

**Comment by mathjazz** [comment]
Fan-tas-tic, r+!

**Pull request merged**